configure.ac

```diff
@@ -208,6 +208,31 @@ if test "$gd_with_LIBFREETYPE" = "yes"; then
   AC_DEFINE([HAVE_FT2BUILD_H], [1], [Define if you have the ft2build.h header.])
 fi
 
+dnl CTL configure tests
```

Comment:
we are using `AC_ARG_ENABLE` because ctl is a feature not a single package, so it should be `--enable-ctl` as `--with-ctl` does not seem appropriate.